scalableminds/webknossos
# Persist volume fallback layer attribute to NML

### URL of deployed dev instance (used for testing):
- https://persistfallbacklayer.webknossos.xyz

### Steps to test:
- create new volume tracing with existing segmentation layer, download, reupload, should still show segmentation layer
- create new volume tracing with new segmentation layer, download, reupload, should still not show segmentation layer
- use `tools/volumeAddFallbackLayer.py` to add the segmentation layer attribute to a volume tracing that lacks it

### Issues:
- fixes #3078 

### When Deploying: 
- [ ] wrote discuss announcement

------
